companies/voiscooterscom.json

```diff
@@ -0,0 +1,43 @@
+{
+    "slug": "voiscooterscom",
+    "relevant-countries": [
+        "de",
+        "es",
+        "pt",
+        "fr",
+        "nl",
+        "ch",
+        "it",
+        "at",
+        "dk",
+        "se",
+        "no",
+        "fi"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "Voi Technology AB",
+    "runs": [
+        "Voi Technology Germany GmbH"
+    ],
+    "address": "Mikael Söderberg\nÖstermalmsgatan 26\n114 26 Stockholm\nSweden",
```

Comment:
Please don't include the name here. We will automatically add an `Attn: Data Protection Officer` (or the translated equivalent) in the generator.